tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -212,30 +212,37 @@ version of the raw dataset will probably go away.
   <!-- ======================== -->
 
   <mksrf_fvegtyp>
-    <!-- Potential Vegetation dataset -->
+    <!-- Potential Vegetation dataset TODO Peter Lawrence? -->
     <entry pft_years="PtVg" >
       <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.noanthro.c20230226/mksrf_landuse_ctsm52_noanthroLUH2_1.c20230226.nc</data_filename>
```

Comment:
@lawrencepj1 I assume we need a TRENDY version of this file to have a complete set.

Speaking of a complete set, should we address the missing hi-res files in this version?

Reply:
Do we have updated PtVg and high resolution datasets?

Reply:
Both of these come up in Erik's review below, so I'm resolving this one.